src/main/java/frc/robot/hardware/MotorController.java

```diff
@@ -19,7 +19,10 @@ public static enum MotorConfig {
         BackLeftModuleDrive(7, 50),
         BackLeftModuleTurn(8, true),
         BackRightModuleDrive(6, 50),
-        BackRightModuleTurn(5, true);
+        BackRightModuleTurn(5, true),
+        //Intake motors
+        IntakeMotor1(13),
```

Comment:
Add a comment starting with TODO, stating that we have to update these IDs to match the real hardware